Testscripts/Linux/utils.sh

```diff
@@ -3586,7 +3586,7 @@ function get_OSdisk() {
 	for driveName in /dev/sd*[^0-9];
 	do
 		# Get the OS disk based on "Linux filesystem" string or BootFlag(*) of a partition
-		fdisk -l $driveName | grep -i "Linux filesystem\|/dev/sd[a-z][0-9]\+[ ]*\*" > /dev/null
+		fdisk -l $driveName 2> /dev/null | grep -i "Linux filesystem\|/dev/sd[a-z][0-9]\+[ ]*\*" > /dev/null
```

Comment:
fdisk -l content is not displayed as it gets parsed to grep.
Are you doing this just to filter out for the exit code?

Otherwise I don't think this is required.

Reply:
If there is an error output from the 'fdiks -l', the return value is not right. Like this:
# fdisk -l /dev/sda
Disk /dev/sda: 64 GiB, 68719476736 bytes, 134217728 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 4096 bytes
I/O size (minimum/optimal): 4096 bytes / 4096 bytes
Disklabel type: dos
Disk identifier: 0x9400d65a

Device     Boot  Start       End   Sectors  Size Id Type
/dev/sda1  *      2048    499711    497664  243M 83 Linux

**Partition 3 does not start on physical sector boundary.**

The return value of the function:
deviceName=Partition 3 does not start on physical sector boundary.

/dev/sdc

In fact, "deviceName=/dev/sdc" is expected.